CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ## Bug fixes:
 * Fix: Ambiguous steps reported wehn definition matches via more than one tag (#95)
+* Fix incorrect reading of binding language (#104)
```

Comment:
I would make it a bit more specific about what is fixed, like "Fix incorrect binding language detection for Visual Studio extension" or something similar.